website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -178,11 +178,11 @@ We don’t recommend mixing and matching icon use; that’s to say, if using an
   <Hds::Dropdown::ListItem::Interactive @text="About Consul" @color="action" @icon="play-circle" />
   <Hds::Dropdown::ListItem::Interactive @text="Why Consul on HCP" @color="action" @icon="link" />
   <Hds::Dropdown::ListItem::Interactive @text="Success story" @color="action" @icon="play-circle" />
-</Doc::ListContainer>
-!!!
+  </Doc::ListContainer>
+  !!!
 
-!!! Dont
-<Doc::ListContainer class="hds-dropdown-list">
+  !!! Dont
+  <Doc::ListContainer class="hds-dropdown-list">
```

Comment:
Pretty sure these do/dont will not work if there's space before the `!!!` delimiter. Can you check?